Rubberduck.Parsing/VBA/IDeclarationFinderProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Rubberduck.Parsing.Symbols;
```

Comment:
I am not attached to the manager names. I just called everything manager that did both provide and modify. Feel free to suggest different names.
As stated in a earlier reply to another comment, I figured out that I can actually make the `RubberduckParserState` implement the `IDeclarationManager` interface. So, the `IDeclarationFinderProvider` could get the `RefreshDeclarationFinder` method and this interface can die.